integration/grid.py

```diff
@@ -68,15 +74,15 @@ class FlogGatherer(object):
     Flog Gatherer process.
     """
 
-    # it would be best to use attr.validators.provides() here with the
-    # corresponding Twisted interface (IProcessTransport,
-    # IProcessProtocol) but that is deprecated; please replace with
-    # our own "provides" as part of
+    # it would be best to use attr.validators.provides() here but that
+    # is deprecated; please replace with our own "provides" as part of
     # https://tahoe-lafs.org/trac/tahoe-lafs/ticket/4056#ticket
```

Comment:
probably can update this comment a bit more than get rid of the ticket link, right?